recipes/wg-blimp/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.9.0" %}
+
+package:
+  name: wg-blimp
+  version: {{ version }}
+
+source:
+  url: https://github.com/MarWoes/wg-blimp/releases/download/v0.9.0/wg-blimp-v0.9.0.tar.gz
+  sha256: 7508b014dab5674f4b49745c487230fdecbb2b065ef08280234bee1e02604f58
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python=3.5
```

Comment:
is there any reason you pin it to 3.5?

Reply:
Changed so 3.5 is only used as minimum. 3.5 is required as minimum because of some python features are used that require at least 3.5. Also, not stating a version seems to cause conflicts (2.7 is too low for snakemake), is this sufficient for defining versions or is there a better way to state the dependency on python >= 3.5?